packages/cardano-services/test/PgBoss/stakePoolBatchDelistHandler.test.ts

```diff
@@ -0,0 +1,53 @@
+import { DataSource } from 'typeorm';
+import { PoolDelistedEntity } from '@cardano-sdk/projection-typeorm';
+import { WorkerHandlerFactoryOptions } from '../../src/PgBoss';
+import { createSmashStakePoolDelistedService } from '../../src/StakePool/HttpStakePoolMetadata/SmashStakePoolDelistedService';
+import { initHandlerTest } from './util';
+import { logger } from '@cardano-sdk/util-dev';
+import { stakePoolBatchDelistHandlerFactory } from '../../src/PgBoss/stakePoolBatchDelistHandler';
+
+jest.mock('../../src/StakePool/HttpStakePoolMetadata/SmashStakePoolDelistedService');
+
+describe('stakePoolBatchDelistHandler', () => {
+  let dataSource: DataSource;
+
+  beforeAll(async () => {
+    const testData = await initHandlerTest();
+    dataSource = testData.dataSource;
+  });
+  it('contain latest delisted pools only', async () => {
+    const expected = [
+      'pool1                                                   ',
```

Comment:
Took a while to understand, probably worth a comment :sweat_smile: 
```suggestion
      // Padded by spaces because pool id length is fixed
      'pool1                                                   ',
```

Reply:
Haha, it took a while for me to find out why my tests were failing!